pages/common/mv.md

```diff
@@ -3,10 +3,14 @@
 > Move or rename files and directories.
 > More information: <https://www.gnu.org/software/coreutils/mv>.
 
-- Move files in arbitrary locations:
+- Move file in arbitrary location:
 
 `mv {{source}} {{target}}`
 
+- Move files into another directory, keeping the filenames:
+
+`mv {{source1}} {{source2}} {{source3}} {{directory}}`
```

Comment:
```suggestion
`mv {{/path/to/source1}} {{/path/to/source2}} {{/path/to/source3}} {{/path/to/target}}`
```

Reply:
Dont commit this until we have had others opinions

Reply:
Wouldn't it be better if it were `/path/to/directory` rather than `/path/to/target` considering that it is required to be a directory when multiple files are passed?
![image](https://user-images.githubusercontent.com/6032823/119063141-a824f180-b9d8-11eb-82bf-fe299bc9f170.png)

Reply:
target_directory?

Reply:
Seems fine too, I'll wait to see what others think.

Reply:
I like `/path/to/target_directory` the best too 👍🏻

Reply:
Why the absolute paths? It also worrks with relative.

Reply:
yeah, works with relatives.

Reply:
I think `target_directory` would be more clear than just `target`

Reply:
I think the only change we should make is this:
```suggestion
`mv {{source1}} {{source2}} {{source3}} {{target_directory}}`
```
since the whole page doesn't use `path/to/`. We can of course fix this in a seperate PR, but as @bl-ue pointed out [here](https://github.com/tldr-pages/tldr/pull/5992#issuecomment-844156753), we should not use PRs to fix things unrelated to the original changes.

Reply:
Feel free to commit Simons change!